docs/reference/formatting.md

```diff
@@ -0,0 +1,66 @@
+<!--[metadata]>
++++
+title = "Output formatting"
```

Comment:
A better title is: "Format command and log output"
I'd put this in the user guide rather than in the reference and I would move it to the `docs/admin` folder

The ability to manipulate the output of commands and logs is something administrators will want to use. The initial material you provide is a start. We need to expand it with more and larger examples.  I'll make an issue for that to track.

@calavera WDYT?